src/components/app.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+import axios from 'axios';
+import SearchForm from './search-form';
+import '../styles/app.scss';
+
+const url = 'https://mcr-codes-weather.herokuapp.com/forecast';
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: '',
+        country: '',
+      },
+      errorMessage: '',
+      searchText: '',
+    };
+  }
+
+
+  componentDidMount() {
+    this.updateCity('Hebden Bridge');
+  }
+
+  updateCity = (city) => {
+    this.setState({ errorMessage: '' });
+    axios.get(`${url}?city=${city}`)
+      .then(response => {
+        this.setState({
+          forecasts: response.data.forecasts,
+          location: response.data.location,
+        });
+      })
+      .catch(err => {
+        this.setState({
```

Comment:
How could you change your error handling to take account of the different types of errors that this request could throw?